docs/GettingStarted.asciidoc

```diff
@@ -264,6 +264,52 @@ Each click point can have an `id`, and if a needle contains multiple click point
 you must pass it to `testapi::assert_and_click` to select which click point
 to use.
 
+=== Configuration
+The different components of openQA read their configuration from the following
+files:
+
+* `openqa.ini`: This is the "web UI" config as services providing the web
+  interface and related services such as the scheduler are configured via that
+  file.
```

Comment:
This makes it sounds like this file would actually be necessary but by default we would not even need it anymore. We could try to start with the  "Locations" section and then put this list together with drop-in configurations but I guess you wanted to explain first configurations applicable for the individual components. How about leaving out the extension and not taking about files yet but call it "configuration components"? Something like



```suggestion
The different components of openQA use the following configuration components:

* `openqa`: This is the "web UI" configuration for services providing the web
  interface and related services such as the scheduler.
```

Reply:
That's bad because there is no "component" that is literally called `openqa`. I would rather explicitly name these configuration files. When I read documentation I usually don't want any abstract bla-bla but concrete information.

I can add a note that the files are not required to exist.

Reply:
For now I only fixed the phrasing and added an explicit note stating that defaults are used if the files are not present.